recipes-connectivity/prplmesh/prplmesh.bb

```diff
@@ -10,25 +10,43 @@ OECMAKE_SOURCEPATH = "${S}/"
 
 SECTION = "net"
 LICENSE = "BSD-2-Clause-Patent.txt"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=7b45146d47e73bcbac1068e5cfc2a9fb"
-DEPENDS = "python-native python-pyyaml-native json-c openssl zeromq"
+# LIC_FILES_CHKSUM = "file://LICENSE;md5=7b45146d47e73bcbac1068e5cfc2a9fb"
```

Comment:
Why comment this out? BTW how did you get this hash to begin with? The LICENSE file has had the 0518 hash since half of April.

Reply:
This is because initial recipe was made on prplfoundation/prplMesh@911392026280d9961823a9359e6cc4f08c9ee0a2 commit